resources/gui/js/apis/timeline.js

```diff
@@ -368,6 +368,11 @@ class TimelineApi extends IApi {
     this._itemsOverview.update(data);
   }
 
+  removeItem(id) {
```

Comment:
What happens if id does not exist?
Do we need to catch that?

Reply:
No, "The method ignores removal of non-existing items, and returns an array containing the ids of the items which are actually removed from the DataSet." (https://ww3.arb.ca.gov/ei/tools/lib/vis/docs/dataset.html#Remove)